### PR TITLE
gotify-server: 2.0.11 -> 2.0.12

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by jonringer** [comment]
i don't think you meant to include the back off vimPlugin logic, can you please rebase ontop of current master

**Comment by doronbehar** [comment]
Thanks for catching that!

**Comment by FRidh** [comment]
Something seems to fail.

File: pkgs/servers/gotify/default.nix [hunk 1]
```diff
@@ -18,10 +18,10 @@ buildGoModule rec {
     owner = "gotify";
     repo = "server";
     rev = "v${version}";
-    sha256 = "0zrylyaxy1cks1wlzyf0di8in2braj4pfriyqa24vipwrlnhvgs6";
+    sha256 = "0pkws83ymmlxcdxadb1w6rmibw84vzhx9xrhxc6b1rjncb80l0kk";
```
Comment (jonringer) [inline pkgs/servers/gotify/default.nix]: the version in this file should get bumped as well, I think you're running into a caching issue if this passes locally

**Comment by doronbehar** [comment]
Right right right... Fixed, thanks!

**Review by Ma27 — APPROVED** [review]

**Pull request merged**